### DISCUSSION TITLE
Getting error while adding in Angular 11

### DISCUSSION THREAD

**Discussion by woodsongem** [nolimits4web/swiper#4772] [Q&A]
Hello All,

please help me to fix below error.

node_modules/swiper/angular/angular/src/swiper.module.d.ts:1:22 - error NG6002: Appears in the NgModule.imports of AppModule, but could not be resolved to an NgModule class.

This likely means that the library (swiper/angular) which declares SwiperModule has not been processed correctly by ngcc, or is not compatible with Angular Ivy. Check if a newer version of the library is available, and update if so. Also consider checking with the library's authors to see if the library is expected to be compatible with Ivy.

1 export declare class SwiperModule {

**Comment by vltansky** [comment]
Can you provide reproduce link?
